packages/webapp/app/layout.tsx

```diff
@@ -0,0 +1,17 @@
+import type { PropsWithChildren, ReactElement } from 'react';
+import React from 'react';
+import Providers from './providers';
+
+export default function RootLayout({
+  children,
+}: PropsWithChildren): ReactElement {
+  return (
+    <html lang="en">
```

Comment:
We want to check if we want to add some top level stuff from `_app` inside of this, eg. metadata, icons, preconnect and so on. Probably most can be added through Next.js `Metadata` APIs

Reply:
Yes. I need to check what is needed, thanks for pointing this!

Reply:
Added ✔️ 

Only not added yet is the theme color because is a longer porting and for now is not needed in funnel